API.md

```diff
@@ -107,13 +107,13 @@ server.views({
         }
       },
       prepare: (options, next) => {
-        options.compileOptions.environment = nunjucks.configure(['node_modules/govuk-frontend', ...options.path])
+        options.compileOptions.environment = nunjucks.configure(['node_modules/@envage/hapi-govuk-question-page', ...options.path])
```

Comment:
This should be as it was - it's including the GOV.UK frontend, which is necessary for this plugin to work.

Reply:
Actually it needs both, I've updated the docs so both are listed.